0.10/README.md

```diff
@@ -170,3 +170,9 @@ Below is an example _package.json_ file with the _main_ attribute and _start_ sc
 
 #### Note:
 `oc rsync` is only available in versions 3.1+ of OpenShift.
+
+#### Define your own "Dev Mode":
+
+The provided `package.json` example includes a `scripts.dev` entry.  You can define your own custom [`npm run`](https://docs.npmjs.com/cli/run-script) scripts in your application's `package.json` file.
+
+This image allows you to select alternate / custom runtime modes by setting the `$NPM_RUN` environment variable to match one of your available [`scripts`](https://docs.npmjs.com/misc/scripts) entries.
```

Comment:
i'd just move this whole section into the `Environment Variables` section, adding an entry for the NPM_RUN env variable.

see this for a better example of how we should be doc'ing env variables the image respects:
https://github.com/openshift/mongodb/tree/master/2.4#environment-variables